src/doc/python_scripting/quickrecipes.rst

```diff
@@ -1090,3 +1090,21 @@ Here is an example using a named color table and ``SetColorControlPoints``:
 
 Available color table names can be found via ``ColorTableNames()``.
 
+Creating an expression that maps materials to values
+----------------------------------------------------
+
+A use case that has come up with some VisIt users is the ability to associated a scalar with material.
+The following example defines a function that creates an expression that maps material numbers to values.
+It take a list of tuples of material number and value.
+The material number of the last tuple is ignored and the value is used for any unspecified materials.
+The expresion generated by calling the function is then used in a Psuedocolor plot.:
+
+.. literalinclude: ../../test/tests/quickrecipes/expressions.py
+    :language: Python
+    :start-after: # mapMaterialsToValues {
+    :end-before: # mapMaterialsToValues }
+
+.. figure::  https://media.githubusercontent.com/media/visit-dav/visit/develop/test/baseline/quickrecipes/expressions/materials_to_values.png
```

Comment:
I am not sure the *figure* directive supports URL image regs. I know the *image* directive does, `.. image::` I guess we can commit and see.

And, since the ref'd image won't be on `develop` until the PR is merged, it is indeed hard to tell 😉